bsp4s/src/main/scala/ch/epfl/scala/bsp/endpoints/Endpoints.scala

```diff
@@ -45,6 +45,8 @@ trait BuildTarget {
   object scalaMainClasses
       extends Endpoint[ScalaMainClassesParams, ScalaMainClassesResult](
         "buildTarget/scalaMainClasses")
+  object jvmTestEnvironment extends Endpoint[JvmEnvironmentParams, JvmEnvironmentResult]("buildTarget/jvmTestEnvironment")
```

Comment:
Should we not have one for running too? e.g. `jvmRunEnvironment`?

Reply:
Can you elaborate on the purpose of that? For the concrete use-case we have at the moment in IntelliJ, we don't have the need for separate run/test environment.

Reply:
I would be fine with `jvmEnvironment`, which can generically handle any kind of `java -classpath ...` invocation

Reply:
It's fine by me if you don't include `jvmRunEnvironment`, but `jvmTestEnvironment` should not be renamed to `jvmEnvironment` because the environment in `run` or `test` could be different

Reply:
fair enough, it can indeed be different in sbt :+1:

Reply:
I think if we have `jvmTestEnvironment` we should have a corresponding `jvmRunEnvironment` or just general `jvmEnvironment`, even if it's not needed for the motivating use case.